frontend/src/static/css/panels.scss

```diff
@@ -90,6 +90,16 @@
 .toolbar {
   cursor: pointer;
   float: right;
+
+  &--multiline {
```

Comment:
actually, is it needed to have this extra `--multiline` class that has a duplicate `float: right` property?

Reply:
This class is to let the links on the toolbar appear on separate lines. May I know what is your suggestion?

Reply:
as in, there's no need for the `&--multiline` right? can just 
```
.toolbar {
    ...
    > a {
        ...
    }
}
```

Reply:
My concern would be that would apply to links on all toolbars. While some cases we might not want the links to appear on separate lines (although for now we don't have any).